x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/details/execution_log_table/rule_duration_format.test.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { shallow } from 'enzyme';
+import React from 'react';
+import { RuleDurationFormat } from './rule_duration_format';
+
+// TODO: Replace snapshot test with base test cases
```

Comment:
Will try to add these frontend jest tests in this PR, but if not will grab in a follow-up post FF.

Reply:
Tests added in https://github.com/elastic/kibana/pull/130072